tests/test_kafka_consumer.py

```diff
@@ -1,12 +1,14 @@
 import os
+import mock
 import pytest
 from consumer.consumer import Consumer
 from kafka.errors import NoBrokersAvailable
 
-KAFKA_HOST = os.environ['KAFKA_HOST']
 
 @pytest.fixture
+@mock.patch.dict(os.environ, {'KAFKA_HOST':'localhost'})
 def consumer():
+    KAFKA_HOST = os.environ['KAFKA_HOST']
```

Comment:
not sure if mock above is required since you simply can set `KAFKA_HOST` to whatewer you want to without messing with environment